### DISCUSSION TITLE
How to prevent the files or folders from being rename or delete?

### DISCUSSION THREAD

**Discussion by manikandanmaks** [coder/code-server#5993] [Q&A]
I have tried  **chmod u-w /path/to/folder** to change a file/folder as readonly, but it's allow me to delete or rename a file/folder.

Kindly help on this @jsjoeio 

Thanks in advance,
Manikandan

**Comment by code-asher** [comment]
Could you elaborate on your full setup?  Even without write permissions on Linux I believe you can rename a directory if you have write permissions to the parent directory.  Also removing write on the parent will still allow you to modify any files in nested directories so maybe you just need to apply them recursively (with `-R`).
